src/components/themed/SettingsTappableRow.js

```diff
@@ -1,12 +1,17 @@
 // @flow
 
 import * as React from 'react'
+import AntDesignIcon from 'react-native-vector-icons/AntDesign'
 import FontAwesome5 from 'react-native-vector-icons/FontAwesome5'
 
 import { type ThemeProps, withTheme } from '../services/ThemeContext.js'
 import { SettingsRow } from './SettingsRow.js'
 
 type OwnProps = {
+  // The icon to show on the right.
+  // Defaults to navigate, which shows an arrow.
+  action?: 'navigate' | 'add' | 'delete' | 'lock' | 'unlock',
```

Comment:
Lock & unlock were meant for the settings scene, but somehow I lost that change when I rebased. Deep, gotta add it back.

Reply:
Done. Branch is updated.